testsuite/MDAnalysisTests/topology/test_top.py

```diff
@@ -201,6 +204,136 @@ class TestPRMParser(TOPBase):
     expected_elems = None
 
 
+class TestPRMChainidParser(TOPBase):
```

Comment:
Sorry to ask for more here, but could we add a quick extra test here that just checks that we do actually have the right chainIDs? It feels like this is the only thing that's missing from all this.